tests/index-test.ts

```diff
@@ -86,7 +86,7 @@ describe('resolve-package-path', function () {
   });
 
   if (require('os').platform() !== 'win32') {
-    describe.only('yarn pnp usage', function () {
```

Comment:
😱 waaaat, we had this `.only` landed in master? 😭

Reply:
ooops!!

Reply:
ya was introduced by https://github.com/stefanpenner/resolve-package-path/commit/448d3a19f9bc5b8b0855f0c50950a0e7937eb4fb